docs/install.md

```diff
@@ -60,7 +60,16 @@ To install Compose, do the following:
 
 ### Install using pip
 
-    $ sudo pip install -U docker-compose
+Compose can be installed from [pypi](https://pypi.python.org/pypi/docker-compose)
+using `pip`.  If you install using `pip` it is highly recommended that you use a
+[virtualenv](https://virtualenv.pypa.io/en/latest/) because many operating systems
+have python system packages that conflict with docker-compose dependencies.
```

Comment:
That's fair, my thinking is this: If someone is familiar with virtualenv, they're just going to do things their way regardless of the steps we outline.  If someone is not familiar with virtualenv, calling out the steps to use it (and have the executable available without having to source the virtualenv) is going to be really helpful to them.

Maybe we can address this by explaining that this example is just a suggestion?